include/xtensor/xaccumulator.hpp

```diff
@@ -64,7 +62,8 @@ namespace xt
     template <class RF, class IF>
     auto make_xaccumulator_functor(RF&& accumulate_func, IF&& init_func)
     {
-        using accumulator_type = xaccumulator_functor<std::remove_reference_t<RF>, std::remove_reference_t<IF>>;
+        using accumulator_type
+            = xaccumulator_functor<std::remove_reference_t<RF>, std::remove_reference_t<IF>>;
```

Comment:
I prefer to have both LHS and RHS on the same line; ig the line is too long, we can put the second argument of `xaccumulator_functor` below the first one (and aligned).